clang/include/clang/Basic/LangOptions.def

```diff
@@ -293,6 +293,7 @@ LANGOPT(
 LANGOPT(SYCLDisableRangeRounding, 1, 0, "Disable parallel for range rounding")
 LANGOPT(SYCLEnableIntHeaderDiags, 1, 0, "Enable diagnostics that require the "
         "SYCL integration header")
+LANGOPT(SYCLAllowVirtualFunctions, 1, 0, "Allow virtual functions calls in code for SYCL device")
```

Comment:
It looks like at least some lines attempt to fit into some limit (like 80 symbols?) and I suggest we do the same for this new option.

Reply:
Done 614537b